app/templates/components/edit-reminder.hbs

```diff
@@ -1,3 +1,5 @@
 {{#link-to 'reminders.reminder'}}
   <button class='save-reminder-btn' {{action 'saveDirtyDiana' on='click'}}>Save</button>
 {{/link-to}}
+
+<button class='undo-btn' {{action 'rollBackTheRock' on='click'}}>Undo</button>
```

Comment:
My fault but I forgot to also request that this button only be active and/or visible if the model has dirty attributes.